apps/client/src/module/dashboard/layout/account-menu.tsx

```diff
@@ -28,62 +31,83 @@ export function AccountMenu() {
               src={user.avatar}
               alt={user.displayName + " avatar"}
             />
-            <span>{user?.displayName}</span>
+            <span className="dark:text-primary-100 text-primary-700 ml-[10px] mr-[8px] text-[13px]">
+              {user?.displayName}
+            </span>
 
-            <Icon icon="arrow-down" />
+            <svg
+              className="h-[18px]"
+              width="24"
+              height="24"
+              fill="#bbbbc2"
+              viewBox="0 0 24 24"
+            >
+              <path d="M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z"></path>
+            </svg>
           </>
         )}
         {!user && "Loading..."}
       </button>
 
       <menu
         className={clsx(
-          "bg-primary-50 dark:bg-primary-500 absolute right-0 top-full grid w-60 transition-[grid-template-rows_padding] duration-300",
-          showDropdown ? "grid-rows-[1fr] py-2" : "grid-rows-[0fr] py-0",
+          "bg-primary-50 dark:bg-primary-900 absolute right-0 top-full z-[101] grid w-[210px] shadow-[0_2px_15px_0_rgba(0,0,0,0.05)] transition-[grid-template-rows_padding] duration-300",
+          showDropdown ? "grid-rows-[1fr] py-[15px]" : "grid-rows-[0fr] py-0",
```

Comment:
:3 About these values such as `[15px]` , Do you think we should also create some standard (???) in our config files? Or just leave it here?

Reply:
I think the burger button icon is fine, it's mentioned in the note

Reply:
yeah, I can confirm the button anima on login page is not working, is that happened before this PR?